benchmarks/imagenet/vitb16/mae.py

```diff
@@ -21,24 +22,25 @@ def __init__(self, batch_size_per_device: int, num_classes: int) -> None:
         self.batch_size_per_device = batch_size_per_device
 
         decoder_dim = 512
-        vit = vit_b_16()
+        vit = vit_base_patch16_224(dynamic_img_size=True)
```

Comment:
Do we have to set `dynamic_img_size=True`? I think MAE uses 16x16 patches with 224px images everywhere.